Baremetal-Blinky/README.md

```diff
@@ -0,0 +1,32 @@
+## Baremetal Blinky example using Mbed OS
+
+This guide reviews the steps required to build and run a baremetal Blinky application on Ti's CC3220SF LaunchXL platform without using DigitalOut class and timers.
+
+Please install [Mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli).
+
+This example retrieves code from https://github.com/ARMmbed/mbed-os-ti-port. It's a private repo, please contact support@mbed.com to request access.
+
+### Import the example application
+
+From the command-line, import the example:
+
+```
+git clone https://github.com/ARMmbed/mbed-os-examples-docs_only.git
+cd Baremetal-Blinky
+mbed config root .
+mbed deploy
```

Comment:
You can shorten this to the following:

```
mbed import https://github.com/ARMmbed/mbed-os-examples-docs_only
cd Baremetal-Blinky
```

`mbed import` takes care of cloning, `mbed config root .`, and `mbed deploy`.

Reply:
Oh boy. Instructions are starting to be made using what was originally meant to be a temporary repo.

I think I would have prefered the README.md be made asif the subdirectory were its own repo.